src/actions/events.js

```diff
@@ -0,0 +1,37 @@
+export const addEvent = ({
+  title = "",
+  description = "",
+  organisator = "",
+  city = "",
+  category = "",
+  image = false,
+  createdAt = 0,
+  startDate = 0,
+  endDate = 0,
+  id = "",
```

Comment:
Are you sure that you need `id` here? The `id` will probably be created on your backend side (i.e. firebase). As I remember, you want to use firebase, so it's redundant. `id` could be needed if you decided to use i.e. IndexedDB.

Reply:
I needed id to verify if edit action works correctly and I will remove it later.